README.md

```diff
@@ -91,6 +91,7 @@ No matter your preferred language, our SDKs provide the tools you need to levera
 | --- | --- |
 | [Apache KIE SonataFlow](https://sonataflow.org) | Apache KIE SonataFlow is a tool for building cloud-native workflow applications. You can use it to do the services and events orchestration and choreography. |
 | [Synapse](https://github.com/serverlessworkflow/synapse) | Synapse is a scalable, cross-platform, fully customizable platform for managing and running Serverless Workflows. |
+| [Lemline](https://github.com/lemline/lemline) | Lemline is a highly scalable runtime running on top of your existing messaging infrastructure. |
```

Comment:
I think we use alphabetical ordering so it should be before Synapse.

Reply:
@geomagilles We indeed ordered it alphabetically, @JBBianchi is right

Reply:
sorry, it should be ok by now